### ISSUE TITLE
Issue in FastBert evaluation

### ISSUE THREAD

**Issue by JulesBelveze** [JulesBelveze/bert-squeeze#32]
There's an issue in the evaluation of `FastBert` when on training stage 1.

The scorer and the way the evaluation report is logged doesn't expect to receive a list of logits (corresponding to the logits of all the ramp layers).
The model probably needs to overrides both the `log_eval_report` and `validation_epoch_end` methods of the parent class.
